notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -440,9 +450,9 @@
         "id": "EpmJq75zXjmT"
       },
       "source": [
-        "### Create Entity Types\n",
+        "### Create the Vertex AI Entity Type resources\n",
         "\n",
-        "Entity types can be created within the Featurestore class. Below, you create the `Users` entity type and `Movies` entity type. Process logs are printed in the output for each cell."
+        "Vertex AI Entity Types can be created within a Vertex AI Feature Store resource. Below, you create the `Users` entity type and `Movies` entity type. Process logs are printed in the output for each cell."
```

Comment:
"You can create entity types within a featurestore. Use the following methods to create the `users` and `movies` entity types. Process logs are printed in the output for each cell."